prince/mca.py

```diff
@@ -47,7 +47,8 @@ def transform(self, X):
         return self.row_coordinates(X)
 
     def plot_coordinates(self, X, ax=None, figsize=(6, 6), x_component=0, y_component=1,
-                         show_row_points=True, row_points_size=10, show_row_labels=False,
+                         show_row_points=True, row_points_size=10, 
+                         row_points_alpha = 0.6, show_row_labels=False,
```

Comment:
Can you remove the whitespace around the `=`?

Reply:
Sure!